File: MockInterview.Backend/MockInterview.BLL/Models/DTOs/UserTokenDto.cs
namespace MockInterview.BLL.Models.DTOs
{
    public class UserTokenDto
    {
        public long Id { get; set; }

        public long? TalentId { get; set; }

        public string Token { get; set; } = null!;

        public DateTime ExpireTime { get; set; }
    }
}

File: MockInterview.Frontend/src/types/userTokenDto.ts
import {IMappable} from "@/types/common";

export interface IUserTokenDto {
    id: number,
    talentId: number,
    token: string,
    expireTime: Date | string
}

export class UserTokenDto implements IMappable<IUserTokenDto> {
    id!: number;
    talentId!: number;
    token!: string;
    expireTime!: Date | string;

    mapFrom(data: IUserTokenDto) {
        this.id = data.id;
        this.talentId = data.talentId;
        this.token = data.token;
        this.expireTime = data.expireTime;
    }
}